Src/Metrics.Tests/Reporting/EventMetricsCleanerTest.cs

```diff
@@ -0,0 +1,212 @@
+﻿
+using System;
+using System.Collections.Generic;
+using FluentAssertions;
+using Metrics.Core;
+using Metrics.MetricData;
+using Metrics.Reporters.Cleaners;
+using Metrics.Utils;
+using Xunit;
+
+namespace Metrics.Tests.Reporting
+{
+    public class EventMetricsCleanerTest
+    {
+        [Fact]
+        public void StartsWithZeroReports()
+        {
+            EventMetricsCleaner.Clear();
```

Comment:
Clear should not be called here.

Reply:
Right you are. Have made all other tests clear after they have run.